scopes/component/component-filters/envs-filter.module.scss

```diff
@@ -3,13 +3,12 @@
   > div {
     line-height: 16px;
     display: flex;
-    // otherwise when the dropdown is open it clashes with the content of the tree
-    z-index: $nav-z-index + 2;
```

Comment:
fixes this (by moving the index to the menu only)

![Screen Shot 2022-06-14 at 11 21 18](https://user-images.githubusercontent.com/5400361/173549738-6696b8ad-8121-4695-bbae-0de8cfc1fed1.png)